### DISCUSSION TITLE
Some Icons don‘t work

### DISCUSSION THREAD

**Discussion by wurstcasesenario** [zauberzeug/nicegui#3247] [Q&A] [answered]
I don‘t know why but some icons I found on [Google Fonts](https://fonts.google.com/icons?icon.set=Material+Icons) don‘t work, or display the wrong way. For example “barcode“ or “barcode_scanner”. Am I just missing some kind of update or aren’t those icons supported by nicegui yet?

**Comment by rodja** [comment]
I'll close this as duplicate of https://github.com/zauberzeug/nicegui/issues/1155

**Comment by falkoschindler** [comment, marked as answer]
Short answer: Since "barcode" and "barcode_scanner" are symbols, you need to use the "sym_" prefix and a variant indicator like "o" for "outlined".
```py
ui.icon('sym_o_barcode')
ui.icon('sym_o_barcode_scanner')
```
<img width="32" alt="Screenshot 2024-06-21 at 17 35 15" src="https://github.com/zauberzeug/nicegui/assets/5767091/551e408d-190e-4273-adbd-2d0882a0a4a2">
